Makefile

```diff
@@ -19,7 +19,7 @@ vet:
 	go vet -v
 
 test:
-	go test -v
+	go test -v -timeout 10s -count 2
```

Comment:
Oooh, *magic* numbers, fun!  With stuff like this I think it is a good idea to briefly mention in the code why a timeout of 10s is necessary and why you picked a count of two.

Reply:
Makes sense. I'll add the following explanation:
```diff
+# Previous bugs have made the tests freeze until the timeout. Golang default
+# timeout for tests is 10 minutes, which is too long, considering current tests
+# can be executed in less than 1 second. Reduce the timeout, so problems can
+# be noticed earlier in the CI.
+TEST_TIMEOUT=10s
+
+# Some tests run with SetTsync(false) and some tests with SetTsync(true). Once
+# the threads are not using the same seccomp filters anymore, the kernel will
+# refuse to use Tsync, causing next tests to fail. This issue could be left
+# unnoticed if the test with SetTsync(false) is executed last.
+#
+# Run tests twice ensure that no test leave the testing process in a state
+# unable to run following tests, regardless of the subset of tests selected.
+TEST_COUNT=2
+
 test:
-       go test -v -timeout 10s -count 2
+       go test -v -timeout $(TEST_TIMEOUT) -count $(TEST_COUNT)
```